protected/humhub/widgets/DirectoryFilters.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2021 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\widgets;
+
+use humhub\components\Widget;
+use humhub\libs\Html;
+use Yii;
+use yii\helpers\ArrayHelper;
+
+/**
+ * DirectoryFilters displays the filters on the directory people/spaces pages
+ *
+ * @since 1.9
+ * @author Luke
+ */
+abstract class DirectoryFilters extends Widget
```

Comment:
Can you please move this class + views to the `ui` module?

Reply:
Commit https://github.com/humhub/humhub/pull/5057/commits/39d6f26750e710a8ef2c9485293bef7b098783b2